server/chat-plugins/poll.ts

```diff
@@ -291,7 +292,9 @@ export const commands: ChatCommands = {
 			if (!this.can('minigame', null, room)) return false;
 			if (supportHTML && !this.can('declare', null, room)) return false;
 			if (!this.canTalk()) return;
-			if (room.minorActivity) return this.errorReply("There is already a poll or announcement in progress in this room.");
+			if (room.minorActivity && cmd !== 'queue') {
+				return this.errorReply("There is already a poll or announcement in progress in this room.");
+			}
```

Comment:
You should probably have the check for "is it possible to queue" here as well. Splitting apart the two checks makes the code less readable.

Reply:
Specifically, the `"There is already a queued activity."` check.